### ISSUE TITLE
I suggest adding props trStyle 

### ISSUE THREAD

**Issue by kozlikov** [AllenFang/react-bootstrap-table#1451]
```
<BootstrapTable data={ data } trClassName={ this.trClassFormat } trStyle={ this.trStyleFormat } >
// ........
</BootstrapTable>
```

How do you like this idea?

**Comment by AllenFang** [comment]
I'll support it in the future, thanks

**Comment by AllenFang** [comment]
@kozlikov fixed on `v3.4.6`, check [this](https://github.com/AllenFang/react-bootstrap-table/blob/master/examples/js/style/tr-style-table.js) for example, 
You can assign a style object to `trStyle` or a function, which will be more customizable.

Thanks

**Comment by kozlikov** [comment]
thanks a lot
